### ISSUE TITLE
Fehler server

### ISSUE THREAD

**Issue by Bandesss** [modelleicher/FS22_simpleIC#35]
[log.txt](https://github.com/modelleicher/FS22_simpleIC/files/10521766/log.txt)
Hallo wir haben folgenden fehler.

**Comment by Bandesss** [comment]
Auch ist uns aufgefallen, dass unsere Maus in der Übersicht Karte verloren geht.
Möchte ungern auf den Mod verzichten

**Comment by Bandesss** [comment]
Gibts hier noch infos ?

**Comment by jason0611** [comment]
Bist Du sicher, dass Du das korrekte Logfile hochgeladen hast? In dem zugehörigen Spielstand ist FS22_simpleIC nämlich nicht vorhanden.

**Comment by Bandesss** [comment]
Doch doch wir haben ihn drauf die Version von Github

**Comment by jason0611** [comment]
Dann passt aber dennoch das Log nicht, darin ist kein FS22_simpleIC enthalten.

**Comment by Bandesss** [comment]
Ok muss ich schauen ok

**Comment by Bandesss** [comment]
So nun die richtige Log
[log.txt](https://github.com/modelleicher/FS22_simpleIC/files/10608741/log.txt)

**Comment by jason0611** [comment]
Ja, jetzt ist es enthalten. Ihr nutzt noch Version 0.9.0.7, aktuell ist Version 0.9.1.0. Bitte testet einmal diese Version.

**Comment by Bandesss** [comment]
Machen wir danke

**Comment by Bandesss** [comment]
Haben die Neue Version und Maus immer noch weg
[log.txt](https://github.com/modelleicher/FS22_simpleIC/files/10609398/log.txt)

**Comment by Bandesss** [comment]
Unsere Maus verschwindet, in der Log ist nicht, was können wir machen ?
[Uploading log.txt…]()

**Comment by Bandesss** [comment]
Gibt's hier noch Hilfe?
